roles/init-conf/tasks/main.yml

```diff
@@ -75,13 +90,34 @@
     - init
     - upstart
 
-- name: Install celery beat systemd service
-  template: src=kernelci-celery-beat.service
-            dest=/etc/systemd/system/kernelci-celery-beat.service
+- name: Install celery systemd services
+  template: src={{ item }}.service
+            dest=/usr/lib/systemd/system/{{ item }}.service
```

Comment:
I was under the impression the `/usr/lib/systemd` path was for system packages, not something manually installed. Is CentOS systemd ran in a different way compared to Debian (user mode maybe)?